cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -263,9 +263,9 @@
     {% compress js %}
     <script>
     {% for js in page.media %}
-       {% set compononet_js_source = component_ondemand_js(js) | trim %}
-       {% if compononet_js_source | length > 0 %}
-            {{ compononet_js_source }}
+       {% set compononent_js_source = component_ondemand_js(js) | trim %}
+       {% if compononent_js_source | length > 0 %}
+            {{ compononent_js_source }}
```

Comment:
The changes in this file are just fixing a typo we noticed.

Reply:
yeah, it's still a typo.

Reply:
Oof, wow :laughing: